pkg/util/common/common_test.go

```diff
@@ -128,5 +131,27 @@ func TestStructToMap(t *testing.T) {
 				},
 			},
 		},
-	}, StructToMap(top))
+	}
+
+	topNilOutput := map[string]interface{}{
+		"name":  "",
+		"value": 0,
+		"mymap": map[string]interface{}{},
+	}
+
+	cases := []struct {
+		name string
+		in   Top
+		out  map[string]interface{}
+	}{
+		{"notNil", top, topOutput},
+		{"nil", topNil, topNilOutput},
+	}
+
+	for _, tc := range cases {
+		t.Run(fmt.Sprintf("%v", tc.name), func(t *testing.T) {
```

Comment:
No need for `Sprintf` here - can just use `tc.name`.

Reply:
will do, thanks